### PR TITLE
Use OpenStreetMap Tiles

### PR DESCRIPTION
**Pull request by lukaswelte**
The referenced komoot tiles will be shut down soon.
To keep a map on the website please switch to public OpenStreetMap tiles

This only changes the example config, likely the actually used config is private - so please switch it there as well :) 

I'm happy to answer any kind of questions.
Thank you

### CODE REVIEW

**Comment by otbutz** [comment]
Please use `tile.openstreetmap.org/{z}/{x}/{y}.png`. see https://github.com/openstreetmap/operations/issues/737

**Comment by lukaswelte** [comment]
@otbutz thanks for reaching out - updated the PR

File: website/config.js.example [hunk 1]
```diff
@@ -1,4 +1,4 @@
 API_URL = 'http://localhost:5001/api/?';
-TILELAYER = 'https://{s}.tile.hosted.thunderforest.com/komoot-2/{z}/{x}/{y}{r}.png';
+TILELAYER = 'https://tile.openstreetmap.org/{z}/{x}/{y}.png;
```
Comment (lonvia) [inline website/config.js.example]: Minor typo here. You deleted a ' too many.

**Comment by lonvia** [comment]
I've updated photon.komoot.io accordingly.

**Pull request merged**